r/_pkgdown.yml

```
@@ -23,6 +23,14 @@ template:
   params:
     bootswatch: cosmo
     ganalytics: UA-107500873-1
+  opengraph:
+    image:
+      src: man/figures/card.png
```

I forget, does this work? If so, you don't need to put a copy here at all.

```suggestion
      src: https://arrow.apache.org/img/arrow-logo_horizontal_black-txt_white-bg.png
```

---

🤦🏻‍♀️  why didn't I think of that? Yes, that's much easier! 😁